src/contacts/view/ContactViewModel.ts

```diff
@@ -46,17 +45,18 @@ export class ContactViewModel {
 	async init(contactListId?: Id, contactId?: Id) {
 		// update url if the view was just opened
 		if (contactListId == null && contactId == null) this.updateUrl()
-		if (this.contactListId) return
+
+		// If we have already initialized the view model with the user's contact list, then select the contact within it
+		if (this.contactListId && typeof contactId === "string") {
```

Comment:
similarly I am trying to understand why `loadAndSelect()` below is not sufficient

Reply:
Good point, I could merge this into the assignment on line 55.
```js 
this.contactListId = this.contactListId
			? this.contactListId
			: assertNotNull(await this.contactModel.getContactListId(), "not available for external users")
```